src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1704,4 +1704,43 @@ public String getTextFromComment(Text commentText) {
         }
         return SanitizationHelper.sanitizeForCsv(comment.toString());
     }
+
+    /**
+     * Checks if bundle contains response from Instructor.
+     *
+     * @return
+     * <ul>
+     * <li>true if the bundle contains response from Instructor</li>
+     * <li>false otherwise</li>
+     * </ul>
+     */
+    public boolean containsResponseFromInstructor() {
+        for (FeedbackResponseAttributes response : responses) {
+            String giverIdentifier = response.giver;
+            // Some parts of the code handles a participant as a Student if participant is both Student and Instructor
+            if (isParticipantIdentifierInstructor(giverIdentifier) && !isParticipantIdentifierStudent(giverIdentifier)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Checks if bundle contains response to Instructor or General.
+     *
+     * @return
+     * <ul>
+     * <li>true if the bundle contains response to Instructor or General</li>
+     * <li>false otherwise</li>
+     * </ul>
+     */
+    public boolean containsResponsesToInstructorOrGeneral() {
+        for (FeedbackResponseAttributes response : responses) {
+            String recipientIdentifier = response.recipient;
+            if (getSectionFromRoster(recipientIdentifier) == Const.NO_SPECIFIC_SECTION) {
```

Comment:
Add comment: `getSectionFromRoster` will return Const.NO_SPECIFIC_SECTION for recipient who is Instrutor or Nobody specific.